JsonParser.h

```diff
@@ -1,13 +1,54 @@
+/**
+ * \class JsonParser
+ *
+ * \brief JsonParser class
+ *
+ * This class declares the JsonParser object, which is used to correctly read the heroes attributes.
+ *
+ * \author NullPointerException
+ *
+ * \version 1.0
+ *
+ * \date 2020/11/01 11:00
+ *
+ */
+
 #ifndef JsonParser_h
 #define JsonParser_h
 #include <map>
 
+//This is the class of the JsonParser
 class JsonParser{
 private:
+    
+    /**
+     * \brief Json file format checking, and cleaning out the blank spaces
+     *
+     * \return Returns a string that has the json files data
+     */
     static std::string WhitespaceCleanerAndFormatChecker(std::string&);
```

Comment:
1. Ennél az egynél várnék egy pontos paraméter leírást, hogy egyértelmű legyen. Pláne, hogy csak típusokkal van deklarálva a headerben, szóval még a változónév sem árulkodik. Key és value is? Akár egyszerre az egész sor?

2. Egy egyszerű leírás is kellene, hogy mire ellenőriz? Hogy idézőjeleknél megfelelő string? Hogy nélkülük megfelelő numerikus érték (vagy boolian vagy null)?